src/components/ActionBar/ActionBar.tsx

```diff
@@ -45,89 +46,77 @@ const ACTIONS = {
+  if (!latest) {
+    return null;
   }
 
-  private renderIconsWithLink(link: string, component: JSX.Element): ReactElement<HTMLElement> {
-    return (
-      <a href={link} target={'_blank'}>
-        {component}
-      </a>
-    );
-  }
+  const { homepage, bugs, dist } = latest;
 
-  private renderActionBar = ({ packageMeta }) => {
-    const { latest } = packageMeta;
+  const actionsMap = {
+    homepage,
+    issue: bugs ? bugs.url : null,
+    tarball: dist ? dist.tarball : null,
+  };
 
-    if (!latest) {
-      return null;
+  const renderList = Object.keys(actionsMap).reduce((component: React.ReactElement[], value, key) => {
+    const link = actionsMap[value];
+    if (link && isURL(link)) {
+      const actionItem: Action = ACTIONS[value];
+      if (actionItem.handler) {
+        const fab = (
+          <Tooltip key={key} title={actionItem['title']}>
+            <Fab
+              /* eslint-disable react/jsx-no-bind */
+              onClick={() => {
+                /* eslint-disable @typescript-eslint/no-non-null-assertion */
+                actionItem.handler!(link);
+              }}
+              size={'small'}>
+              {actionItem['icon']}
+            </Fab>
+          </Tooltip>
+        );
+        component.push(fab);
+      } else {
+        const fab = <Fab size={'small'}>{actionItem['icon']}</Fab>;
+        component.push(
+          <Tooltip key={key} title={actionItem['title']}>
+            <>{renderIconsWithLink(link, fab)}</>
+          </Tooltip>
+        );
+      }
     }
+    return component;
+  }, []);
 
-    const { homepage, bugs, dist } = latest;
-
-    const actionsMap = {
-      homepage,
-      issue: bugs ? bugs.url : null,
-      tarball: dist ? dist.tarball : null,
-    };
-
-    const renderList = Object.keys(actionsMap).reduce((component: React.ReactElement[], value, key) => {
-      const link = actionsMap[value];
-      if (link && isURL(link)) {
-        const actionItem: Action = ACTIONS[value];
-        if (actionItem.handler) {
-          const fab = (
-            <Tooltip key={key} title={actionItem['title']}>
-              <Fab
-                /* eslint-disable react/jsx-no-bind */
-                onClick={() => {
-                  /* eslint-disable @typescript-eslint/no-non-null-assertion */
-                  actionItem.handler!(link);
-                }}
-                size={'small'}>
-                {actionItem['icon']}
-              </Fab>
-            </Tooltip>
-          );
-          component.push(fab);
-        } else {
-          const fab = <Fab size={'small'}>{actionItem['icon']}</Fab>;
-          component.push(
-            <Tooltip key={key} title={actionItem['title']}>
-              <>{this.renderIconsWithLink(link, fab)}</>
-            </Tooltip>
-          );
-        }
-      }
-      return component;
-    }, []);
+  if (renderList.length > 0) {
+    return (
+      <List>
+        <ActionListItem alignItems={'flex-start'} button={true}>
```

Comment:
You can remove  `button={true}`, not needed here.

Reply:
Sure